### ISSUE TITLE
The changes in loss values during the pre-training period of the space model.

### ISSUE THREAD

**Issue by ychuest** [ZhuJiwei111/SPACE#4]
Dear authors,

Hello. I trained the model on a single A100 (80G) GPU and observed that the values of loss_human and loss_mouse fluctuate significantly, which may be due to a small batch size on a single card. I would like to know what the loss curves looked like during your actual training process, as well as how the loss changed during model evaluation. If possible, could you please provide some reference data or loss curves for comparison?

Thank you very much!

Below are the loss curves I observed during training on a single GPU.

<img width="864" height="516" alt="Image" src="https://github.com/user-attachments/assets/a1795bc0-a45f-462f-8f6a-1e3d065284f6" />

<img width="864" height="516" alt="Image" src="https://github.com/user-attachments/assets/af0e595a-9bf3-4a47-91f9-b885ece4b326" />

**Comment by ZhuJiwei111** [comment]
Hi, I also think the loss fluctuation is caused by the total_batch_size you set being too small. If you only have a single GPU, try increasing the `gradient_accumulation_steps` to achieve total_batch_size = `nGPUs` * `per_device_batch_size` * `gradient_accumulation_steps` = 64.  
  
Of course, I'd be happy to share our loss curve with you. You can view it at:  
https://api.wandb.ai/links/zhujiwei-/0tkfi6mt

**Comment by ychuest** [comment]
Thank you so much. This has provided me with great insight. I will close this issue.
